src/account/default.ts

```diff
@@ -827,6 +886,13 @@ export class Account extends Provider implements AccountInterface {
     ) as Promise<AccountInvocations>;
   }
 
+  /**
+   * Retrieves the StarkNet name associated with a given address.
```

Comment:
```suggestion
   * Retrieves the Starknet name associated with a given address.
```

If I don't point it out here in review, someone will make a PR xD